### PR TITLE
Wip lesson

### LINKED ISSUE #1: needs content
[Refer to Outline](https://github.com/learn-co-curriculum/sinatra-fwitter-1-mvc/blob/master/README.md)

### PR DESCRIPTION
**Pull request by ipc103**
fixes #1  - needs review.

### CODE REVIEW

**Comment by AnnJohn** [comment]
@dfenjves can you review this? Will set a deadline for 9/11 but feel free to adjust to 9/15 if needed.

**Comment by ipc103** [comment]
thanks - @dfenjves let me know if you want me to shorten this, it's pretty
long.

On Thu, Sep 10, 2015 at 1:39 PM, AnnJohn notifications@github.com wrote:

> @dfenjves https://github.com/dfenjves can you review this? Will set a
> deadline for 9/11 but feel free to adjust to 9/15 if needed.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/learn-co-curriculum/sinatra-fwitter-1-mvc/pull/2#issuecomment-139321121
> .

## 

```
  Ian Candy
  Pre-College Instructor
```

**Comment by AnnJohn** [comment]
@dfenjves Deadline now for 9/15

File: README.md [hunk 1]
```diff
@@ -1,25 +1,351 @@
...
+
+Fork and clone this repository to get started!
 
 ## Instructions
 
-Create make the tests pass Heroku lab. API Nokogiri guest speaker belongs_to RESTful binder.ply. Url Rails slack it to me. Asset pipeline Feelings Friday puts "woof" API open source.
+To begin, we'll be building out the Sinatra file structure from scratch. While we do, we'll talk about what each file is doing and the different responsibilites of our application. We could, if we wanted to, write our entire application in one file, but it would be very difficult to read and debug. Keeping things organized is called "seperation of concerns" and is an important concept when prgoramming. Let's start building out our file structure. 
+
+### 1). Gemfile
+
```
Comment (dfenjves) [inline README.md]: Are students forking and cloning this, or building from scratch?

**Comment by dfenjves** [comment]
Looks great @ipc103. Made a few small changes. Closes #1

**Pull request merged**